Create index.html

MarieMatsumoto opened the pull request:
課題提出用

MarieMatsumoto committed:
Create index.html

MarieMatsumoto committed:
Update index.html

修正して再度提出（グローバルナビゲーション部分に<div>追加、リンク先に"#"追加、クラス名追加・変更）

MarieMatsumoto committed:
Update index.html

クラス名、<div>追加

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,294 @@
+    <title>NRIの強み：ビジネスモデルの特徴 | 企業情報 | 野村総合研究所(NRI)</title>
+    <link rel="stylesheet" href="html5reset-1.6.1.css">
+    <link rel="stylesheet" href="/nri-html/style.css">
+    <link rel="stylesheet" href="/Users/marie/Desktop/3rd-week/nri-html/style.css">
```
今回はcssは特に見ないので良いですが、このパスだと他の人のpcで見たときにcssが見れないですね😵

MarieMatsumoto replied:
ローカル環境のまま上げていました。CSS課題のほうで反映させるようにいたします。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,294 @@
+<body>
+    <div class="position-lock">
+        <header>
+            <div class="flexbox-1">
```
class名はもう少し意味のある名前をつけてあげた方が良いですね😌

MarieMatsumoto replied:
すみません、適当すぎました…。変更します。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,294 @@
+        <div class="key-visual">
+            <div class="key-text">
+                <p>NRIの強み</p>
+                <h2>NRIのデジタルビジネス戦略</h2>
```
h1タグが見当たらなかったです。(あったらごめんなさい)
h1がなくてh2があるのは構成的におかしいので、確認をお願いします！

MarieMatsumoto replied:
すみません、H1が抜けていました。ここをH2からH1に変更します。

MarieMatsumoto committed:
Update index.html

MarieMatsumoto reviewed (commented):
スタイルシートの参照元URLを一旦空欄に、クラス名を変更、見出しの番号変更

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,293 @@
+                        <path d="M46.16,18.26c3.06,4.64,7.28,10.1,11.91,16H47.53C44,28.7,39.24,22.11,35.39,17.33c5.16-1.15,9.35-3.31,9.35-7.47,0-3-2.16-5.36-7.33-5.36-2.05,0-3.76.14-5.16.14-.14,4.11-.19,8.37-.19,12.62,0,5.8.32,11.59.54,17H25A252.11,252.11,0,0,0,6.56,11.39c-.1,5.94.06,14.75.71,22.86H0c.24-5.4.55-11.19.55-17S.24,5.67,0,.27H8.78A248.16,248.16,0,0,0,25.71,21.49c0-5.86-.12-13.83-.73-2 ...
+                    </svg>
+                </a>
+                <div class="global-navi">
```
ナビゲーションはナビゲーション用のタグがあるのでそっちを使ってあげたほうが良いですね

MarieMatsumoto replied:
修正しました。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,293 @@
+                </div><!-- 謎のリンク -->
+
+                <div class="strength">
+                    <h4>NRIの強み</h4>
```
ここが h4 の場合はこれの前に h3 がある必要があります。

MarieMatsumoto replied:
ここをH3に修正します。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,293 @@
+
+        <div class="corporate-container">
+            <div class="corporate-info">
+                <h3>企業情報</h3>
```
階層構造的にここが h3 の場合は ` <h2 class="main-h2">NRIのデジタルビジネス戦略</h2>` の下にコンテンツが存在しているイメージになるのですが、そんなことはないと思うので、h3 ではない方が良さそうです。

この辺はメインのコンテンツから外れていることもあるので、見出しタグは使わなくても良さそうですね 😄

MarieMatsumoto replied:
すべてpタグに変更します。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,293 @@
+    </div><!-- SNS -->
+    <div class="footer-box">
+        <div class="fb-mini">
+            <h4><a href="#">ナレッジ・インサイト</a></h4>
```
この辺も見出しタグは使わなくても良いかと思います！

ykyk1218 commented:
追加で！
head タグ内に discriptionやkeywords、OGPのmetaタグの追加をお願いします！

MarieMatsumoto committed:
Update index.html

metaタグ（description, keywords, OGP）追加、「NRIの強み」部分をh3に変更、企業情報以下の見出しをすべて削除しpに変更、企業情報をh3からpに変更したことによってborder-top, border-bottomが効かなくなってしまうのでdivで囲むように変更

ykyk1218 commented:
良さそうです！！
LGTM 👍
